stackage_snapshot.json

```diff
@@ -1,6 +1,6 @@
 {
```

Comment:
Also needs an update on Windows.

Reply:
bump: done
windows: right....

Reply:
Using a windows ad-hoc machine,  updated: `stackage_snapshot_windows.json`
First time I've managed to get this to work!
Thanks @garyverhaegen-da for walking me through the setup.
I will record the steps in the working on `ghc-lib` document...